src/host/ut_host/CodepointWidthDetectorTests.cpp

```diff
@@ -38,18 +38,6 @@ class CodepointWidthDetectorTests
         VERIFY_IS_TRUE(widthDetector.IsWide(emoji));
     }
 
-    TEST_METHOD(CanExtractCodepoint)
-    {
-        CodepointWidthDetector widthDetector;
-        for (const auto& data : testData)
-        {
-            const auto& expected = std::get<0>(data);
-            const auto& wstr = std::get<1>(data);
-            const auto result = widthDetector._extractCodepoint({ wstr.c_str(), wstr.size() });
-            VERIFY_ARE_EQUAL(result, expected);
-        }
-    }

```

Comment:
This is already covered by the `CanGetWidths` test. `_extractCodepoint` doesn't exist anymore.